### PR TITLE
Add support for ordering

### PR DESCRIPTION
**Pull request by UnwrittenFun**

### CODE REVIEW

File: src/Types.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Observable, Subject } from 'rxjs/Rx';
 export type PropertyType = string | number | Date | boolean;
 export type AdvancedPropertyType = PropertyType | { [s: string]: PropertyType };
 export type Range = { min: PropertyType, max: PropertyType };
+export type Order = 'desc' | 'asc' | '';
```
Comment (connor4312) [inline src/Types.ts]: Imo we should disallow empty ordering and require it to always be explicit

Comment (SimonSchick) [inline src/Types.ts]: There is also `trending`, `rising` and `fresh`.

Comment (SimonSchick) [inline src/Types.ts]: But that's a special case for channel listing...

Comment (UnwrittenFun) [inline src/Types.ts]: Thats the reason for the empty string.

Comment (SimonSchick) [inline src/Types.ts]: Well yes, but wouldn't it make sense to check if asc/desc is missing that the value may only be one of the three?

It's a little bit specific/hacky but would safeguard us there.

Comment (UnwrittenFun) [inline src/Types.ts]: That seems far too specific for my liking

Comment (SimonSchick) [inline src/Types.ts]: Perhaps being able to define custom orderings on a model might be better then?
If you don't want to do it I can do it some time later.

**Pull request merged**